qemu/tests/bridge_qinq.py

```diff
@@ -288,8 +288,8 @@ def compare_host_guest_md5sum(name):
 
         # scp file to guest with L2 vlan tag
         file_size = int(params.get("file_size", "4096"))
-        host_path = params.get("host_path", "/tmp/transferred_file")
-        guest_path = params.get("guest_path", "/tmp/transferred_file")
+        host_path = params.get("host_path", "/var/tmp/transferred_file")
```

Comment:
From the issue itself, the problem is no enough space in guest os.   Normally, host resource is enough for the file size 4G i think.

Should we use the temporary path of the automation framework ? In this way,  tmp file can be cleaned automatically.
Otherwise,a step to clean tmp filed is needed.

And the step to clean tmp file is guest is must no matter change host_path or not, thanks!

Reply:
Because there is a place to compare the `md5` value, I directly changed the path of the two to the same. In addition, I added a command to delete files from the guest. 
@yanan-fu Please help me review it again, thanks a lot.

Reply:
@yanglei-rh  
1. It is better to put the clean step in `finally`
2. Miss clean step for host since you do not use tmp dir of our automation framework.

By the way, IMO, use different path for host and guest is not a blocker for md5 comparing, just update function `compare_host_guest_md5sum` to accept two path is ok.